src/core/Remixer.ts

```diff
@@ -168,60 +168,60 @@ class Remixer {
 
   /**
    * Adds a string Variable to array of variables with optional callback
-   * @param  {string}              key            The key of the Variable.
-   * @param  {string}              defaultValue   The initial default value of the variable.
-   * @param  {string[]}            possibleValues The optional array of available items for the variable.
-   * @param  {IVariableCallback}   callback       The callback method to be invoked
-   *                                              when the Variable is updated.
+   * @param  {string}              key             The key of the Variable.
+   * @param  {string}              defaultValue    The initial default value of the variable.
+   * @param  {string[]}            limitedToValues The optional array of available items for the variable.
```

Comment:
The description could be improved, what about: "When set, the variable can only be set to one of these values", or "The optional array of allowed values"?